docs/streams/upgrade-guide.html

```diff
@@ -34,7 +34,9 @@ <h1>Upgrade Guide and API Changes</h1>
     </div>
 
     <p>
-        If you want to upgrade from 1.0.x to 1.1.0 you don't need to make any code changes as the public API is fully backward compatible.
+        If you want to upgrade from 1.0.x to 1.1.0 you don't need to make any code changes as the public API is fully backward compatible in Java 7.
```

Comment:
I think we should move the Java 7 bit to the start of the sentence. i.e, `If you are using Java 7 then ... `